### PR TITLE
Add contribution guidelines and reorg community sections

### LINKED ISSUE #260: Add OSS general contribution guidelines

### PR DESCRIPTION
**Pull request by guzman-raphael**
Depends on #257, #261

### Added
- Added `Community` section. PR #259
- Added previously discussed contribution guidelines document in `Community > Contribute`. (#260) PR #259

### Changed
- Moved `Introduction > Publications` to `Community > Publications` section. PR #259.
- Moved `Introduction > Community` to `Community > Engagements` section. PR #259.

### Removed
- `Introduction > Contribute`. PR #259
- `Introduction > Issues`. PR #259

Note: ~~This PR has a dependency on us properly adjusting the DNS CNAME record from `testdocs.datajoint.io` to `docs.datajoint.io`.~~ Updating the DNS record is no longer a dependency to merge this PR since merging will not trigger a new release. We can safely merge when ready, perform the update to DNS record, and follow with the new release.

Community view
image

Community > Contribute view
image

LaTex View
image

Redirected from search
image

Fix #260

### CODE REVIEW

File: contents/community/02-Contribute.rst [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+Contribute
+==========
+Thank you for your interest in contributing! 🤝
+
+To help keep everyone in alignment and coordinated in the community effort, we've created this document. It serves as the contribution guidelines that outline how open-source software development is to be conducted. Any software development that makes reference to this can be assumed to adopt the policies outlined below. We've structured it in FAQ format to make it easier to digest. Feel free review the questions below to determine any specific policy.
+
+Which issue should I contribute towards?
+----------------------------------------
+There is primarily 3 things to consider when looking to contribute.
```
Comment (dimitri-yatsenko) [inline contents/community/02-Contribute.rst]:
```suggestion
There are three primary things to consider when looking to contribute:
```

File: contents/community/02-Contribute.rst [hunk 2]
```diff
@@ -0,0 +1,96 @@
+.. progress: 1.0 100% Raphael
+
+.. _contribute:
+
+Contribute
+==========
+Thank you for your interest in contributing! 🤝
+
+To help keep everyone in alignment and coordinated in the community effort, we've created this document. It serves as the contribution guidelines that outline how open-source software development is to be conducted. Any software development that makes reference to this can be assumed to adopt the policies outlined below. We've structured it in FAQ format to make it easier to digest. Feel free review the questions below to determine any specific policy.
```
Comment (dimitri-yatsenko) [inline contents/community/02-Contribute.rst]:
```suggestion
To help keep everyone in alignment and coordinated in the community effort, we've created this document. It serves as the contribution guideline that outlines how open-source software development is to be conducted. Any software development that makes reference to this document can be assumed to adopt the policies outlined below. We've structured the guideline in a FAQ (frequently asked questions) format to make it easier to digest. Feel free review the questions below to determine any specific policy.
```

File: contents/community/02-Contribute.rst [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+Which issue should I contribute towards?
+----------------------------------------
+There is primarily 3 things to consider when looking to contribute.
+
+- **Availability**: Simply if anyone is currently working on a fix. This is represented by who is ``assigned``. Issues that are ``unassigned`` mean that there is no one yet working on resolving the issue.
```
Comment (dimitri-yatsenko) [inline contents/community/02-Contribute.rst]:
```suggestion
- **Availability**: An indication of whether anyone is currently working on a fix for the given issue. Availability is indicated by who is ``assigned``. Issues that are ``unassigned`` mean that there is no one yet working on resolving the issue and the issue is available for someone to work on. If an issue has been assigned, then any additional work on that issue should be coordinated with the assignee. 
```

File: contents/community/02-Contribute.rst [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+    1. No syntax errors
+    2. No integration errors
+    3. No style errors e.g. PEP8, etc.
+    4. Similar or better code coverage
+
+- Additional documentation to reflect new feature or behavior introduced.
+- Necessary updates to the changelog following `Keep a Changelog <https://keepachangelog.com/en/1.0.0/>`_ convention.
+- A contributor should not approve or merge their own PR.
+- Reviewer suggestions or feedback should not be directly committed to a branch on contributor's fork. A less intrusive way to collaborate would be for the reviewer to PR to the contributor's fork/branch that is associated with the main PR currently in review.
```
Comment (dimitri-yatsenko) [inline contents/community/02-Contribute.rst]:
```suggestion
- Reviewer suggestions or feedback should not be directly committed to a branch on a contributor's fork. A less intrusive way to collaborate would be for the reviewer to PR to the contributor's fork/branch that is associated with the main PR currently in review.
```

File: contents/community/02-Contribute.rst [hunk 5]
```diff
@@ -0,0 +1,96 @@
+.. progress: 1.0 100% Raphael
+
+.. _contribute:
+
+Contribute
+==========
+Thank you for your interest in contributing! 🤝
+
```
Comment (dimitri-yatsenko) [inline contents/community/02-Contribute.rst:8]: Perhaps add a sentence describing the core development team (DataJoint Neuro), welcoming contributions from others. Perhaps something like:

> The principal maintainer of DataJoint and associated tools is the DataJoint Neuro team (Vathes Inc.) The pronouns "we" and "us" in this guideline refer to the principal maintainers. We invite reviews and contributions of the open-source software. We compiled these guidelines to make this work clear and efficient.

**Review by dimitri-yatsenko — CHANGES_REQUESTED** [review]

File: contents/community/02-Contribute.rst (hunk 1 above)
Comment (guzman-raphael) [reply contents/community/02-Contribute.rst]: Addressed.

File: contents/community/02-Contribute.rst (hunk 2 above)
Comment (guzman-raphael) [reply contents/community/02-Contribute.rst]: Addressed.

File: contents/community/02-Contribute.rst (hunk 4 above)
Comment (guzman-raphael) [reply contents/community/02-Contribute.rst]: Addressed.

File: contents/community/02-Contribute.rst (hunk 5 above)
Comment (guzman-raphael) [reply contents/community/02-Contribute.rst:8]: Addressed.

File: contents/community/02-Contribute.rst (hunk 3 above)
Comment (guzman-raphael) [reply contents/community/02-Contribute.rst]: Addressed.

**Review by dimitri-yatsenko — APPROVED** [review]

**Pull request merged**